### PR TITLE
Change revapi to check against 7.5.0.Final

### PR DESCRIPTION
**Pull request by MarianMacik**
@sutaakar Changes to revapi in KIE Server including removing revapi for swagger module, since it is not needed there.

depends on kiegroup/droolsjbpm-build-bootstrap#775

### CODE REVIEW

**Comment by MarianMacik** [comment]
Jenkins retest this

File: kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm/src/build/revapi-config.json [hunk 1]
```diff
@@ -27,82 +27,174 @@
...
+          "annotation": "@io.swagger.annotations.ApiOperation(value = \"Replaces existing query definition or registers new if not exists in the system with given queryName\", response = org.kie.server.api.model.definition.QueryDefinition.class, code = 201)",
+          "attribute": "response",
+          "oldValue": "java.lang.Void.class",
+          "newValue": "org.kie.server.api.model.definition.QueryDefinition.class",
+          "package": "org.kie.server.remote.rest.jbpm",
+          "classSimpleName": "QueryDataResource",
+          "methodName": "replaceQueryDefinition",
+          "elementKind": "method",
+          "justification": "JBPM-7103 - KIE Server custom queries exposed as DataSet editor in workbench"
+        },
```
Comment (ge0ffrey) [inline kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm/src/build/revapi-config.json]: This adds things too, is that intended?

**Comment by ge0ffrey** [comment]
jenkins retest this

**Comment by ge0ffrey** [comment]
jenkins retest this (somehow the build was aborted...) Note that it was green 2 days ago, I just wanted to have 100% certaintly

**Comment by MarianMacik** [comment]
Indeed it was, but Maria's PR #1484 was merged so I had to update it. Build should be green now.

**Comment by ge0ffrey** [comment]
@mswiderski There are 12 failing unit tests, but I suppose they have nothing to do with this PR. Are you OK that we merge this PR?

```
Test Result (12 failures / +12)

    org.kie.server.integrationtests.jbpm.rest.ProcessServiceRestOnlyIntegrationTest.testProcessDefinitionWhichBelongsToAContainer[0: JAXB KieServicesConfiguration{transport=REST, serverUrl='http://localhost:40415/kie-server-services/services/rest/server'}]
    org.kie.server.integrationtests.jbpm.rest.ProcessServiceRestOnlyIntegrationTest.testProcessVariablesHistoryWhichBelongsToAContainer[0: JAXB KieServicesConfiguration{transport=REST, serverUrl='http://localhost:40415/kie-server-services/services/rest/server'}]
    org.kie.server.integrationtests.jbpm.rest.ProcessServiceRestOnlyIntegrationTest.testProcessVariablesWhichBelongsToAContainer[0: JAXB KieServicesConfiguration{transport=REST, serverUrl='http://localhost:40415/kie-server-services/services/rest/server'}]
    org.kie.server.integrationtests.jbpm.rest.ProcessServiceRestOnlyIntegrationTest.testProcessWhichBelongsToAContainer[0: JAXB KieServicesConfiguration{transport=REST, serverUrl='http://localhost:40415/kie-server-services/services/rest/server'}]
    org.kie.server.integrationtests.jbpm.rest.ProcessServiceRestOnlyIntegrationTest.testAbortWorkItemWhichBelongsToAProcess[0: JAXB KieServicesConfiguration{transport=REST, serverUrl='http://localhost:40415/kie-server-services/services/rest/server'}]
    org.kie.server.integrationtests.jbpm.rest.ProcessServiceRestOnlyIntegrationTest.testNodeInstancesWhichBelongsToAProcess[0: JAXB KieServicesConfiguration{transport=REST, serverUrl='http://localhost:40415/kie-server-services/services/rest/server'}]
    org.kie.server.integrationtests.jbpm.rest.ProcessServiceRestOnlyIntegrationTest.testProcessDefinitionWhichBelongsToAContainer[1: JSON KieServicesConfiguration{transport=REST, serverUrl='http://localhost:40415/kie-server-services/services/rest/server'}]
    org.kie.server.integrationtests.jbpm.rest.ProcessServiceRestOnlyIntegrationTest.testProcessVariablesHistoryWhichBelongsToAContainer[1: JSON KieServicesConfiguration{transport=REST, serverUrl='http://localhost:40415/kie-server-services/services/rest/server'}]
    org.kie.server.integrationtests.jbpm.rest.ProcessServiceRestOnlyIntegrationTest.testProcessVariablesWhichBelongsToAContainer[1: JSON KieServicesConfiguration{transport=REST, serverUrl='http://localhost:40415/kie-server-services/services/rest/server'}]
    org.kie.server.integrationtests.jbpm.rest.ProcessServiceRestOnlyIntegrationTest.testProcessWhichBelongsToAContainer[1: JSON KieServicesConfiguration{transport=REST, serverUrl='http://localhost:40415/kie-server-services/services/rest/server'}]
```

**Pull request merged**